app/src/main/java/it/niedermann/owncloud/notes/manageaccounts/ManageAccountsActivity.java

```diff
@@ -37,22 +38,36 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         List<LocalAccount> localAccounts = db.getAccounts();
 
         adapter = new ManageAccountAdapter((localAccount) -> SingleAccountHelper.setCurrentAccount(getApplicationContext(), localAccount.getAccountName()), (localAccount) -> {
-            db.deleteAccount(localAccount);
-            for (LocalAccount temp : localAccounts) {
-                if (temp.getId() == localAccount.getId()) {
-                    localAccounts.remove(temp);
-                    break;
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
```

Comment:
We have a [`DeleteDialogAlertBuilder`](https://github.com/stefan-niedermann/nextcloud-notes/blob/master/app/src/main/java/it/niedermann/owncloud/notes/branding/BrandedDeleteAlertDialogBuilder.java) class which colors the primary button red - Please use this one instead of the default `AlterDialog.Builder`.